scrapers/javlibrary.yml

```diff
@@ -21,7 +24,7 @@ movieByURL:
       - b47w.com/
     scraper: movieScraper
 xPathScrapers:
-  sceneScraper:
+  sceneScraper by URL:
```

Comment:
this should be left as it was
```yaml
  sceneScraper:
```
why was it changed ?

Reply:
euh idk, maybe i mess up with something in github 😢